cmd/search.go

```diff
@@ -54,33 +53,28 @@ func (o *SearchOptions) Complete(f Factory, args []string) (err error) {
 // Run executes the search command
 func (o *SearchOptions) Run() (err error) {
 
-	if o.Reindex {
-		printInfo(o.Out, "building index...")
-		done := false
-		if err = o.SearchRequests.Reindex(&core.ReindexSearchParams{}, &done); err != nil {
-			return err
-		}
-		printSuccess(o.Out, "reindex complete")
-	}
+	// TODO: add reindex option back in
 
-	p := &repo.SearchParams{
-		Q:      o.Query,
-		Limit:  30,
-		Offset: 0,
+	p := &core.SearchParams{
+		QueryString: o.Query,
+		Limit:       100,
+		Offset:      0,
 	}
-	res := []repo.DatasetRef{}
+	results := &[]core.SearchResult{}
 
-	if err = o.SearchRequests.Search(p, &res); err != nil {
+	if err = o.SearchRequests.Search(p, results); err != nil {
 		return err
 	}
 
 	switch o.Format {
 	case "":
-		for i, ref := range res {
-			printDatasetRefInfo(o.Out, i, ref)
+		fmt.Printf("showing %d results for '%s'\n", len(*results), o.Query)
```

Comment:
After our recent refactoring, all output should be routed to `o.Out`, so this should be:
```golang
printInfo(o.Out, "showing %d results for '%s'", len(results), o.Query)
```
printInfo will add in the newline for you :)